example/worker_section.conf

```diff
@@ -0,0 +1,25 @@
+<system>
+  workers 8
+  root_dir /path/fluentd/root
+</system>
+
+<source> # top-level sections works on all workers in parallel
+  @type forward
+  port 24224
+</source>
+
+<match all> # this section works on all workers too
+  @type stdout
+  <inject>
+    worker_id_key worker_id
+  </inject>
+</match>
+
+<worker 0> # this section works only on first worker process
```

Comment:
This configuration doesn't use the `<worker>` section feature to execute plugins which doesn't support multi process workers.